index.html

```diff
@@ -1932,6 +1927,43 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code>splashscreens</code> member
+        </h3>
+        <p>
+          A <dfn>splash screen</dfn> is a graphical representation (usually an
+          image on a background color) that is presented as the end user before
+          the user agent is able render content. A splash screen is intended to
+          convey to the end-user that the web application is being prepared for
+          them in the background.
+        </p>
+        <p>
+          When showing a <a>splash screen</a>, a user agent MUST announce to
+          assistive technologies that the web application is being loaded in
+          the background.
```

Comment:
This general statement of intent looks good to me. Is this intended to mean the UA automatically announces this? I'm assuming in its current form, an app can't use the splash screen to show additional information (e.g. short status/progress messages of what it's loading/doing - e.g. "reticulating splines 67%" or similar ? Otherwise this would also need to be taken into account (and at that stage, it's more likely that authors will want/need to control what exactly is exposed to AT, e.g. using an `aria-live` region...